infrastructure/webhook_stack/events.tf

```diff
@@ -0,0 +1,12 @@
+resource "aws_cloudwatch_event_bus" "document_unpublish_events" {
+  name = "prismic-document-unpublish-events"
+}
+
+resource "aws_cloudwatch_event_rule" "document_unpublish_events" {
+  name           = "prismic-document-unpublish-events"
+  description    = "Webhook events from Prismic triggered by document unpublish"
+  event_bus_name = aws_cloudwatch_event_bus.document_unpublish_events.name
+  event_pattern = jsonencode({
+    detail-type : ["document-unpublish"]
```

Comment:
Am I right in thinking this will filter for `document-unpublish` events, and that's a value that comes from Prismic?

Reply:
Sort of - Prismic does not tell consumers of webhooks whether the documents have been published or unpublished; this `document-unpublish` value comes from the custom header that I have forced us to add.